packages/persistent-merkle-tree/src/node.ts

```diff
@@ -251,7 +242,7 @@ export class LeafNode extends Node {
 
     // number spans 2 h values
     else if (uintBytes === 8) {
-      if (value === Infinity && clipInfinity) {
+      if (value === Number.POSITIVE_INFINITY && clipInfinity) {
```

Comment:
😢 can we go back to `Infinity` here?  Same with the other "global" JS stuff that was namespaced to `Number` like `parseInt`.

Reply:
Reverted that complete commit earlier, don't know how it reaches back in here. I will clean commit history for this branch.